Gruntfile.js

```diff
@@ -64,3 +64,6 @@ module.exports = function(grunt) {
 
   grunt.registerTask('default', ['clean', 'sass', 'copy', 'babel']);
 };
+
```

Comment:
Please remove - it's good practice to keep change requests as small as possible.